cfme/automate/explorer/method.py

```diff
@@ -209,9 +209,10 @@ def is_displayed(self):
 
 
 class Method(BaseEntity, Copiable):
-    ICON_NAME = VersionPick({
-        Version.lowest(): 'product-method',
-        '5.9': 'fa-ruby',
+
+    ICON_NAME_MAP = VersionPick({
+        Version.lowest(): {'inline': 'product-method'},
+        '5.9': {'inline': 'fa-ruby', 'playbook': 'vendor-ansible'}
     })
```

Comment:
This will not work, VersionPick by itself can only live on the widgets and views only ...